### ISSUE TITLE
Items rendering above the runic altar getting dark

### ISSUE THREAD

**Issue by cabralRodrigo** [Vazkii/Botania#1039]
When the altar is fully covered with block, the items rendering above the altar get really dark.

Screenshots:
[Normal Runic Altar](http://imgur.com/qInjPUY)
[Covered Runic Altar](http://imgur.com/ZCyjUAg)

I'm using Botania version `r1.6-174` on the Regrowth pack version `0.7.4`

**Comment by MrKunji** [comment]
It's a vanilla lighting issue, also, if you can't reproduce on the latest version (191 ATM) it is not a valid issue.

**Comment by cabralRodrigo** [comment]
Here:
[Normal Runic Altar](http://imgur.com/IGasGAY)
[Covered Runic Altar](http://imgur.com/UnRDWSi)

Also, I know if the colorless grass top is a vanilla bug also or a Botania Bug.

I'm using Botania version `r1.6-191`.

**Comment by Entoarox** [comment]
The runic altar as a non-full block inherits its lighting from nearby blocks, further, you are completely obscuring the block space it is in, did you really expect that just because it renders things outside the block space that those items would remain lit?

TLDR; Vanilla bug, keep at least one side of the altar block space exposed to air.

**Comment by TheWhiteWolves** [comment]
It's a vanilla thing so not much we can do about it.
